CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-karma ChangeLog
 
+## 6.3.0 - 2024-xx-xx
```

Comment:
I recommend major bump to avoid triggering failures in user's libs that weren't there before. Let them upgrade to see them and deal with them on their own time.

```suggestion
## 7.0.0 - 2024-xx-xx
```